.github/workflows/test-past-releases.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         sct_version: [ "6.5", "6.4", "6.3", "6.2", "6.1", "6.0", "5.8", "5.7", "5.6", "5.5", "5.4", "5.3.0", "5.2.0", "5.1.0" ]
-        os: [ ubuntu-latest, macos-latest, windows-2019 ]
+        os: [ ubuntu-latest, macos-latest, macos-15, windows-2019, windows-2025 ]
```

Comment:
Everything appears to be working OK on these new platforms! :tada:

The only catch is `windows-2025` on some older releases. There seems to be 1 failing test for the `seg_sc_t2star` model: https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/14473727158/job/40594082475#step:10:463

But, given that we've retired this model anyway, I don't think it's high-priority at all. Meaning, we can proceed as expected with the v7.0 release.

Reply:
I don't really have a solution for this failing model, so I've added a bit of a hack fix to skip the test for `task == 'seg_sc_t2star'` on Windows in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4856/commits/dc37c5eab2f4acfdc248cb2842091412f97df070.

Pretty sure this works for all versions of SCT, but it took a bit of trial and error: https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/15258537463 (famous last words, it's still a WIP right now)